src/ngx_http_lua_args.c

```diff
@@ -14,6 +14,11 @@
 #include "ngx_http_lua_args.h"
 #include "ngx_http_lua_util.h"
 
+#define NGX_HTTP_LUA_FFI_POST_ARGS_OK 0
+#define NGX_HTTP_LUA_FFI_POST_ARGS_EMPTY 1
+#define NGX_HTTP_LUA_FFI_POST_ARGS_UNREAD 2
+#define NGX_HTTP_LUA_FFI_POST_ARGS_IN_FILE 3
```

Comment:
I'd rather not introduce new FFI constants unless absolutely necessary. If every FFI C function introduce its own error codes, it would be a huge amount of such constants in the top-level namespace.